bench/README.md

```diff
@@ -1,60 +1,104 @@
 
 ```sh
 ../examples/tutorial/download_cifar.py /path/to/save/dataset
 ```
 
-### CIFAR10 Binary Dataset for Query Execution
-The C++ benchmark works by sending CIFAR10 query vectors to the container serving the trained SKLearn model. To achieve this, the **binary dataset** is required. It can also be obtained from [https://www.cs.toronto.edu/~kriz/cifar.html](https://www.cs.toronto.edu/~kriz/cifar.html).
 
-## Optional Configuration Files
-The following benchmark attributes can be loaded via a JSON configuration file:
-- **cifar_data_path**: The path to a **specific binary data file** within the CIFAR10 binary dataset with a name of the form `data_batch_<n>.bin`. (`data_batch_1.bin`, for example)
-- **num_threads**: The number of threads of execution
-- **num_batches**: The number of batches of requests to be sent by each thread
-- **batch_size**: The number of requests to be sent in each batch
-- **batch_delay_millis**: The per-thread delay between batches, in milliseconds
+## Spin up a model container to query
+
+The benchmarking tool will send requests through Clipper to a model. You'll want to spin up this model in a container that implements Clipper's Model-Container RPC interface.
+
+You can also use one of our predefined model-container scripts:
+
+- If you want to have the model serving off your machine
+
+  - [`bench/setup_noop_bench.sh`](https://github.com/ucbrise/clipper/tree/develop/bench/setup_noop_bench.sh) runs the [`noop_container`](https://github.com/ucbrise/clipper/blob/develop/containers/python/noop_container.py)
+  - [`bench/setup_sum_bench.sh`](https://github.com/ucbrise/clipper/tree/develop/bench/setup_sum_bench.sh) runs the [`sum_container`](https://github.com/ucbrise/clipper/blob/develop/containers/python/sum_container.py)
+  - [`bench/setup_sklearn_bench.sh`](https://github.com/ucbrise/clipper/tree/develop/bench/setup_sklearn_bench.sh) runs the [`sklearn_cifar_container`](https://github.com/ucbrise/clipper/blob/develop/containers/python/sklearn_cifar_container). If you wish to use this option, remember to download the CIFAR10 python dataset and run
+  ```sh
+  ./setup_bench.sh <path_to_cifar_python_dataset>
+  ```
+where `<path_to_cifar_python_dataset>` is the path to the **directory** containing a parsed CIFAR10 CSV data file with name `cifar_train.data`.
+
+- If you want the model serving off a Docker container, you'll need to build its image. We have some Dockerfiles that define images for some model-containers.
+  - Create the Docker images for the [`noop_container`](https://github.com/ucbrise/clipper/blob/develop/containers/python/noop_container.py)  and [`sum_container`](https://github.com/ucbrise/clipper/blob/develop/containers/python/sum_container.py) by running `./bin/build_bench_docker_images.sh`.
+  - Run `docker run <image_id>` on the same host that the benchmarking tool will be run from.

```

Comment:
Make a note that what the models will be named for each model, so the user knows what name and version to put in the benchmark JSON config.